modules/vpc-network/variables.tf

```diff
@@ -60,13 +60,18 @@ variable "secondary_cidr_subnetwork_spacing" {
 }
 
 variable "log_config" {
-  description = "The logging options for the subnetwork flow logs."
+  description = "The logging options for the subnetwork flow logs. See https://www.terraform.io/docs/providers/google/r/compute_subnetwork.html for more information and examples."
```

Comment:
Maybe also mention that setting to `null` will disable flow logging?

Reply:
Yeah let me add it there as well. 👍